c/include/ml-api-service.h

```diff
@@ -46,15 +47,211 @@ extern "C" {
 typedef void *ml_service_h;
 
 /**
- * @brief Destroys the given service handle.
- * @details If given service handle is created by ml_service_launch_pipeline(), this requests machine learning agent daemon to destroy the pipeline.
+ * @brief Callbacks for the events from machine learning service.
+ * @details Note that the buffer on new_data callback may be deallocated after the return and this is synchronously called.
+ *           Thus, if you need the data afterwards, copy the data to another buffer and return fast. Do not spend too much time in the callback.
+ * @since_tizen 9.0
+ */
+typedef struct {
+  void (*new_data) (ml_service_h handle, const char *name, const ml_tensors_data_h data, void *user_data); /**< Called when new data is processed from machine learning service. */
+  void (*event) (ml_service_h handle, int event, void *event_data, void *user_data); /**< Called when new event is occurred from machine learning service. */
+} ml_service_callbacks_s;
```

Comment:
Agreed, thanks for your suggestion.
We are now preparing Tizen API process as you know.
Let's talk about the event callback and its params on offline discussion, to handle new-data for inference and other events from offloading service.

Reply:
I agreed with @niley7464 's suggestion.
Instead of the structure type event callback, how about this unified form?
ml_service can be passed using the user_data, so I think it's not needed.
And let's use information handle considering later scalability
```
void (*event) (int event_type, ml_information_h data, void *user_data);
```

Reference: Tizen guide
```
3.5.1 Callback function prototype MUST be based on the template
The template is:
typedef void (*{module_prefix}_{actual_name}_cb)(event_type event, {event details}, void *user_data);
```

Reply:
I updated this PR - added event callback.
Please review the changes again.